arangod/Cluster/FollowerInfo.cpp

```diff
@@ -105,11 +105,6 @@ Result FollowerInfo::add(ServerID const& sid) {
         _failoverCandidates = nextCandidates;  // will cast to std::vector<ServerID> const
       }
     }
-#ifdef DEBUG_SYNC_REPLICATION
```

Comment:
if `DEBUG_SYNC_REPLICATION` is removed here, do we still need to handle it in our CMakeLists.txt file?
Or can it be removed from there as well?

Reply:
I think there are other places as well.

Reply:
fine